google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -316,6 +316,73 @@ def _create_load_subparser(parser):
   return load_parser
 
 
+def _create_pipeline_subparser(parser):
+  pipeline_parser = parser.subcommand('pipeline', 'Load from GCS, query, and extract data from GCS '
+                                                  'into a BigQuery table. If creating a new table, '
+                                                  'a schema should be specified in YAML or JSON in '
+                                                  'the cell body, otherwise the schema is inferred '
+                                                  'from existing table.')
+
+  # common arguments
+  pipeline_parser.add_argument('-b', '--billing', type=int, help='BigQuery billing tier')
+  # TODO(rajivpb): I've naively made this a common param. Any objections?
+  pipeline_parser.add_argument('-nc', '--nocache', help='Don\'t use previously cached results',
+                               action='store_true')
+
+  # load arguments
+  pipeline_parser.add_argument(
+      '-lm', '--load_mode', help='One of create (default), append or overwrite for loading data',
+      choices=['create', 'append', 'overwrite'], default='create')
+  pipeline_parser.add_argument('-lf', '--load_format', help='The format of the source file in GCS',
+                               choices=['json', 'csv'], default='csv')
+  pipeline_parser.add_argument('--skip',
```

Comment:
--skip_lines?

Reply:
But, load uses just 'skip' and I want to be consistent:
https://github.com/googledatalab/pydatalab/blob/master/google/datalab/bigquery/commands/_bigquery.py#L305